### PR TITLE
update the version to support 7.0

### PR DESCRIPTION
**Pull request by mobilesuitzero**

### CODE REVIEW

**Review by Joel-Cripps — APPROVED** [review]

File: azure/7.0/ha-port1-mgmt/terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Change to your own azure environment
+subscription_id = "<fill your subscription id>"
```
Comment (jamie-pate) [inline azure/7.0/ha-port1-mgmt/terraform.tfvars:2]: Probably should be renamed tfvars.example, with the tfvars file in .gitignore

File: azure/7.0/ha-port1-mgmt/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Deployment of a FortiGate-VM (BYOL/PAYG) Cluster on the Azure
```
Comment (jamie-pate) [inline azure/7.0/ha-port1-mgmt/README.md:1]: Is is necessary to make a copy in a new folder for each fortigate version? Or can they just be branches (with instructions in the readme)


Seems like a lot of extra effort to review and maintain multiple versions this way.

Maybe a future version could get rid of all the sub directories?

Depends on how incompatible the 7.x version is compared to the 6.4 i guess

**Review by jamie-pate — CHANGES_REQUESTED** [review]

File: azure/7.0/ha-port1-mgmt/README.md (hunk 2 above)
Comment (mobilesuitzero) [reply azure/7.0/ha-port1-mgmt/README.md:1]: 7.x will have some different topology in comparing to 6.x.  hence, that's why the new folder for now.

File: azure/7.0/ha-port1-mgmt/terraform.tfvars (hunk 1 above)
Comment (Joel-Cripps) [reply azure/7.0/ha-port1-mgmt/terraform.tfvars:2]: I don't know why we would put this in gitignore. But if you leave them blank they should work as inputs if that is what you want. Probably just depends on the end-user.

**Pull request merged**